api/provision/consume-provisioning-job.test.ts

```diff
@@ -0,0 +1,101 @@
+import { handler } from "./consume-provisioning-job";
+import { Context, APIGatewayProxyEvent } from "aws-lambda";
+import { mockClient } from "aws-sdk-client-mock";
+import { sqsClient } from "../../utils/aws-sdk/sqs";
+import { DeleteMessageCommand, ReceiveMessageCommand } from "@aws-sdk/client-sqs";
+import { requestContext } from "./start-provisioning-job.test";
+import { ApiSuccessResponse, ErrorStatusCode, OtherErrorResponse, SuccessStatusCode } from "../../utils/response";
+
+export const mockReceiveMessageResponse = {
+  $metadata: {
+    httpStatusCode: 200,
+    requestId: "74b3f95",
+    extendedRequestId: undefined,
+    cfId: undefined,
+    attempts: 1,
+    totalRetryDelay: 0,
+  },
+  Messages: [
+    {
+      MessageId: "b5353c",
+      ReceiptHandle: "AQC0e6b=",
+      MD5OfBody: "b03",
```

Comment:
Did you have to calculate these MD5s?  Or just update value after test failure?

Reply:
No I didn't calculate the MD5s, but got them from previous function executions. I shorted it down so it doesn't look like a secret and cause a false positive.